dags/utils/knack.py

```diff
@@ -19,8 +19,13 @@ def get_date_filter_arg(should_replace_monthly=False, **context):
     Returns:
         Str or None: the -d flag and ISO date string or None
     """
+
     today = now()
-    prev_start_date = context.get("prev_start_date_success") or today.isoformat()
+    prev_start_date = (
+        context.get("prev_start_date_success").isoformat() or today.isoformat()
+    )
```

Comment:
@frankhereford i think this needs a change, because ` context.get("prev_start_date_success").isoformat()` fails with `TypeError` when it is `None`. 

i like that you added the call to `isoformat()` in the return statement, so i think that all that's needed is to change this line to:
```python
 prev_start_date = context.get("prev_start_date_success") or today
```

Reply:
Great catch, thank you. 💯 https://github.com/cityofaustin/atd-airflow/pull/231/commits/97460297ed77ade0bf7d2a5556dd99e832522d90